app/assets/json/translations/locale-en.json

```diff
@@ -653,7 +653,7 @@
             "READ_FULL_LIST": "Read <a href='{{url}}' target='_blank'>our full list of requirements</a> for more details.",
             "STEP_1": "Haven't owned property in the past 3 years",
             "STEP_2": "Meet our <a href='{{url}}' target='_blank'>income requirements</a>",
-            "STEP_3": "Complete a <a href='{{url}}' target='_blank'>homebuyers’ education workshop</a>",
+            "STEP_3": "Complete <a href='{{url}}' target='_blank'>homebuyer education</a>",
```

Comment:
I wonder if this would look better if we call it "First-time homebuyer education" instead of just "homebuyer education"